content/docs/2.5/scalers/newrelic.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "New Relic"
+availability = "2.5+"
+maintainer = "Community"
+description = "Scale applications based on NewRelic NRQL"
+layout = "scaler"
+go_file = "newrelic_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `New Relic` trigger that scales based on a New Relic metric.
+
+```yaml
+triggers:
+  - type: new-relic
+    metadata:
+      # Required: Account - Subaccount to run the query on
+      Account: 1234567
+      # Required: QueryKey - Api key to connect to New Relic
+      QueryKey: "NRAK-xxxxxxxxxxxxxxxxxxxxxxxxxxx"
+      # Optional: nrRegion - Region to query data for
+      Region: "US"
+      # Required: metricName
+      metricName: "duration"
+      # Required: nrql
+      nrql: "SELECT average(duration) from Transaction where appName='SITE' TIMESERIES"
+      # Required: threshold
+      threshold: 100
+      # Optional: nrLogLevel
+      LogLevel: "info"
```

Comment:
Nit - Some have camel casing, others pascal casing

Reply:
All scalers uses camel case with the first letter being lowercase.

Reply:
Pr filed to move them all to CamelCase and updated documentation.